Bottom: 6cfa2c0d9934aa49210231a7244eb4abe3f89a08
Top:    cc627a383bca7302d2c50b2a8884ebed0e3d1b99
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 16:28:11 +0000

Refresh of refactor-to-use-step_id-not

---

diff --git a/src/hid/common/edge3d.c b/src/hid/common/edge3d.c
index 263f986..35a8846 100644
--- a/src/hid/common/edge3d.c
+++ b/src/hid/common/edge3d.c
@@ -2,6 +2,7 @@
 #include <stdlib.h>
 #include <glib.h>
 
+#include "step_id.h"
 #include "edge3d.h"
 
 edge_info *
diff --git a/src/hid/common/face3d.c b/src/hid/common/face3d.c
index 0926001..5f1df0a 100644
--- a/src/hid/common/face3d.c
+++ b/src/hid/common/face3d.c
@@ -1,6 +1,7 @@
 #include <glib.h>
 #include <stdbool.h>
 
+#include "step_id.h"
 #include "quad.h"
 #include "contour3d.h"
 #include "appearance.h"
diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 0197118..34ffceb 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -5,6 +5,7 @@
 
 #include <glib.h>
 
+#include "step_id.h"
 #include "quad.h"
 #include "vertex3d.h"
 #include "contour3d.h"
diff --git a/src/hid/common/vertex3d.c b/src/hid/common/vertex3d.c
index 1704b2b..74c286c 100644
--- a/src/hid/common/vertex3d.c
+++ b/src/hid/common/vertex3d.c
@@ -1,5 +1,7 @@
+#include <glib.h>
 #include <stdlib.h>
 
+#include "step_id.h"
 #include "vertex3d.h"
 
 static int global_vertex3d_count;
diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 015d575..6e8c5d3 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -15,6 +15,7 @@
 #include "gui-pinout-preview.h"
 #include "pcb-printf.h"
 
+#include "hid/common/step_id.h"
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
 #include "hid/common/contour3d.h"
diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 41c287d..5237c4f 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -4,6 +4,7 @@
 
 #include <glib.h>
 
+#include "hid/common/step_id.h"
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
 #include "hid/common/contour3d.h"
